SnapshotTests/SecureConversationsConfirmationScreenLayoutTests.swift

```diff
@@ -19,10 +19,9 @@ class SecureConversationsConfirmationScreenLayoutTests: SnapshotTestCase {
         viewController.view.frame = UIScreen.main.bounds
 
         assertSnapshot(
-            matching: viewController.view,
-            as: .image,
-            named: self.nameForDevice(),
-            record: true
+            matching: viewController,
+            as: .image(on: .iPhone8(.landscape)),
```

Comment:
Is it ok in case we test Snapshots with iPhone 13?

Reply:
That was a mistake. I just wanted to try it out if landscaping works and I forgot to remove it. I will be adding tickets for landscape snapshot tests, as they have proven to be very valuable

Reply:
fixed [added37](https://github.com/salemove/ios-sdk-widgets/pull/769/commits/added37eaff957187641f955d426c2d01d511261)